winml/lib/Api/LearningModelSessionOptions.h

```diff
@@ -31,6 +31,14 @@ struct LearningModelSessionOptions : LearningModelSessionOptionsT<LearningModelS
 
   bool GetIntraOpThreadSpinning();
 
+  const std::vector<winrt::hstring>&
```

Comment:
Same thing here, return a span if possible (I'm not sure if this module easily has access to the GSL).

Reply:
done